codalab/worker/worker_run_state.py

```diff
@@ -205,7 +215,16 @@ def mount_dependency(dependency, shared_file_system):
             #   dependency_path:docker_dependency_path:ro
             docker_dependencies.append((dependency.parent_path, dependency.docker_path))
 
+        log_bundle_transition(run_state.bundle.uuid, run_state.stage, RunStage.PREPARING, '')
```

Comment:
This will result in  "...from <state> to PREPARING due to". We should modify the util function to check if reason exists or not.

Reply:
Like check if the reason exists, then we append the reason; otherwise just truncate the last two words?